modules/ai-adding-worker-nodes-to-cluster.adoc

```diff
@@ -0,0 +1,303 @@
+<path_to_ssh_pub_key> :: is the public SSH key required to access the host. If you do not set this value, you cannot access the host while in discovery mode.
+<image_type> :: is the ISO image type, either `full-iso` or `minimal-iso`.
+--
+
+.. Post the `$INFRA_ENV_REQUEST` to the link:https://api.openshift.com/?urls.primaryName=assisted-service%20service#/installer/RegisterInfraEnv[/v2/infra-envs] API and set the `$INFRA_ENV_ID` variable:
++
+[source,terminal]
+----
+$ INFRA_ENV_ID=$(curl "$API_URL/api/assisted-install/v2/infra-envs" -H "Authorization: Bearer ${JWT_TOKEN}" -H 'accept: application/json' -H 'Content-Type: application/json' -d "$INFRA_ENV_REQUEST" | tee /dev/stderr | jq -r '.id')
+----
++
+.Example output
+[source,terminal]
+----
+{
+    "cluster_id": "84bdd5d1-ebeb-4ebe-ad35-b5336dccf36f",
+    "cpu_architecture": "x86_64",
+    "created_at": "2022-07-06T12:41:38.116442Z",
+    "download_url": "https://api.openshift.com/api/assisted-images/images/a9a691df-6c76-43da-8a56-5b40c1378d1a?arch=x86_64&image_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2NTcxMjU2OTgsInN1YiI6ImE5YTY5MWRmLTZjNzYtNDNkYS04YTU2LTViNDBjMTM3OGQxYSJ9.I42L11XmQ5vVy2hQrvWhGUY-cXm3H_KWTFEEL3iX5xY&type=minimal-iso&version=4.11",
+    "email_domain": "example.com",
+    "expires_at": "2022-07-06T16:41:38.000Z",
+    "href": "/api/assisted-install/v2/infra-envs/a9a691df-6c76-43da-8a56-5b40c1378d1a",
+    "id": "a9a691df-6c76-43da-8a56-5b40c1378d1a",
+    "kind": "InfraEnv",
+    "name": "example-cluster-name",
+    "openshift_version": "4.11",
+    "org_id": "1979710",
+    "proxy":
+    {},
+    "pull_secret_set": true,
+    "ssh_authorized_key": "ssh-ed25519 AAAAC3NzaC1lZDI1NTE3AAAAILKa2wHPqXLntE/ZuiAzDfOp0fZurwV8VPSNvVHv2goR example_user@example.com",
+    "type": "minimal-iso",
+    "updated_at": "2022-07-06T12:41:38.134203Z",
+    "user_name": "example_user"
+}
+----
+
+. Get the URL of the discovery ISO for the cluster worker node by running the following command:
++
+[source,terminal]
+----
+$ curl -s "$API_URL/api/assisted-install/v2/infra-envs/$INFRA_ENV_ID" -H "Authorization: Bearer ${JWT_TOKEN}" | jq -r '.download_url'
+----
++
+.Example output
+[source,terminal]
+----
+https://api.openshift.com/api/assisted-images/images/41b91e72-c33e-42ee-b80f-b5c5bbf6431a?arch=x86_64&image_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2NTYwMjYzNzEsInN1YiI6IjQxYjkxZTcyLWMzM2UtNDJlZS1iODBmLWI1YzViYmY2NDMxYSJ9.1EX_VGaMNejMhrAvVRBS7PDPIQtbOOc8LtG8OukE1a4&type=minimal-iso&version=4.11
+----
+
+. Download the ISO:
++
+[source,terminal]
+----
+$ curl -L "https://api.openshift.com/api/assisted-images/images/41b91e72-c33e-42ee-b80f-b5c5bbf6431a?arch=x86_64&image_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2NTYwMjYzNzEsInN1YiI6IjQxYjkxZTcyLWMzM2UtNDJlZS1iODBmLWI1YzViYmY2NDMxYSJ9.1EX_VGaMNejMhrAvVRBS7PDPIQtbOOc8LtG8OukE1a4&type=minimal-iso&version=4.11" --output rhcos-live-minimal.iso
```

Comment:
suggesting to add -s flag to avoid download progress prints
i.e 
curl -L -s "https://api...."